runtime/src/bank.rs

```diff
@@ -1221,14 +1223,47 @@ impl Bank {
             .map(|(accs, tx)| match accs {
                 (Err(e), hash_age_kind) => (Err(e.clone()), hash_age_kind.clone()),
                 (Ok((accounts, loaders, _rents)), hash_age_kind) => {
```

Comment:
Convert to and from `RefCell<Account>`.  We could move this line down to accounts_db but that's a change/optimization that can happen later.  Doing so is non-trivial but doable.